extensions/spark/kyuubi-spark-authz/src/main/resources/table_command_spec.json

```diff
@@ -2056,6 +2056,30 @@
     "fieldExtractor" : "LogicalPlanQueryExtractor"
   } ],
   "uriDescs" : [ ]
+}, {
+  "classname" : "org.apache.spark.sql.delta.commands.OptimizeTableCommand",
+  "tableDescs" : [ {
+    "fieldName" : "tableId",
```

Comment:
I couldn't found the field named tableId

Reply:
In old version. https://github.com/delta-io/delta/blob/v2.2.0/core/src/main/scala/org/apache/spark/sql/delta/commands/OptimizeTableCommand.scala

Reply:
I think we shall target the latest delta release first, then backward compatibilities for elderlies.

Reply:
Updated

Reply:
Thanks